test/Test/Database/LSMTree/StateMachine.hs

```diff
@@ -604,7 +606,7 @@ instance ( Show (Class.TableConfig h)
     -- Snapshots
     CreateSnapshot ::
          C k v b
-      => Maybe Errors
+      => Maybe (Either SilentCorruptions Errors)
       -> R.SnapshotLabel -> R.SnapshotName -> Var h (WrapTable h IO k v b)
       -> Act h ()
```

Comment:
Would be good to add a `Tag` for snapshot corruption so we can see from the test output that corruption happens (or not)